### DISCUSSION TITLE
Naming the session document

### DISCUSSION THREAD

**Discussion by ceramicv** [TheBlewish/Automated-AI-Web-Researcher-Ollama#25] [Ideas]
Rather than naming the results file as 'research_session_1.txt", "research_session_2.txt" etc why not ask the LLM to create a filename based on the research topic that you provide. Limit it to say 16 characters, that way you can tell what the research was actually about rather than the generic name.

For example I asked ChatGPT to suggest a filename based on my topic of "Capabilities and features of microsoft intune for managing ios devices", and it came back with "Intune_iOS_Mgmt"

**Comment by synth-mania** [comment]
Sounds like an interesting idea. Maybe then we could have a subfolder that it puts all the research_session documents in.

**Comment by TheBlewish** [comment]
Hey not a bad idea, if you wanna go ahead and code it in, and it works after I test i'll merge a pull request if you make it and it doesn't break the program!
